source/plugin/api-versions.rst

```diff
@@ -0,0 +1,81 @@
+============
+API-Versions
+============
+
+This page explains which API versions exists and to which Minecraft version their implementations belong to.
+
++-------------+--------------+----------------+-------------------------------------------+
+| API-Version | Release Date | End of Support | Known Implementations (Minecraft Version) |
++=============+==============+================+===========================================+
+| *Planned*   | ---          | --             | * *SpongeForge (1.13.x)*                  |
+| *8.0.0*     |              |                | * *SpongeVanilla (1.13.x)*                |
++-------------+--------------+----------------+-------------------------------------------+
+| 7.1.0       | 2018-09-06   | Late 2019      | * SpongeForge (1.12.2)                    |
```

Comment:
I'm going to quibble with the date format, as it varies globally whether month or day comes first. Can we use something less ambiguous, like "Jan 1, 2018" or similar?

Reply:
I used [ISO_8601](https://en.wikipedia.org/wiki/ISO_8601) for the format. 

This total mess is just because of the (few) English (and a few other languages) speaking people out there. 😈  

I will change it, although with a sad mood.